### PR TITLE
Add an acronym exercise for the bash language in exercism.io

### PR DESCRIPTION
**Pull request by netserf**
Adding my first programming exercise to exercism.io.

### CODE REVIEW

**Comment by Smarticles101** [comment]
Looks good! the build failed because you haven't put acronym into the config.json file. If you have any questions, just ask!

**Comment by netserf** [comment]
Sorry about that and thanks for the tip. I'll take a look.

**Comment by netserf** [comment]
Okay, I have a few things to learn about git still. I hope this is better now.

File: config.json [hunk 1]
```diff
@@ -173,8 +173,18 @@
       ],
       "unlocked_by": "hamming",
       "uuid": "03808072-0228-6580-7a9b-cc709a8a5b8c0e600b9"
-    }
+    },
+    {
+      "core": false,
```
Comment (NobbZ) [inline config.json]: If it is not `core`, it needs to be `unlocked_by` something.

Comment (netserf) [reply config.json]: Thank you for the feedback Nobbz. Could you point me to the docs on config.json? I guessed at the requirements, but I should really understand it if I'm to add more exercises in the future.

Comment (NobbZ) [reply config.json]: Well, I was told today that this is in fact not an issue. If the key is missing, it is implied to be unlocked by the very first exercise of the track.

More info: exercism/configlet#96

Comment (netserf) [reply config.json]: I read the [exercism/configlet#96](https://github.com/exercism/configlet/issues/96). I can't say I understand the requirements of config.json completely, but as long as I've met the minimum requirements that should work for now. If this isn't documented anywhere, where would be the best place be to add the docs?

**Comment by Smarticles101** [comment]
@netserf Do you want to estimate a difficulty and position in the track along with a good choice for it to be `unlocked_by` or not if you don't see an obvious exercise to unlock it. If not, I can open an issue for whichever of these (or both) you decide not to do.

Also, recently we've added `skips` to exercise test suites. Could you add a `skip` to the first line of every test function with the first one commented out?

Sorry for such a late response 😬  and thanks for the hard work 👍

**Comment by netserf** [comment]
@Smarticles101, sorry it took me a bit to circle back around to this. 
- Estimate a difficulty 
  Calling it a 1 (i.e. not very difficult)
- Position in the track
  Is this the relative position in the config.json file? I'll put it after leap. Please let me know if I'm not understanding this correctly.
- unlocked_by
  Done
- add skip to the first line of every test function with the 1st one commented out
  Done

**Comment by Smarticles101** [comment]
@netserf yep, that's exactly what I was looking for, thanks! Sorry it took me a few days to get back to this. :)

**Comment by netserf** [comment]
Thanks for walking me through the process.  :-)

**Pull request merged**